Resources/Textures/Clothing/Head/Hats/chefhathead.rsi/meta.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Modified from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e",
+  "size": {
+    "x": 32,
+    "y": 64
```

Comment:
just use the offset in prototype

Reply:
I'm unsure how to do this

Reply:
check the plunger

Reply:
I'll try to implement this as a space saving measure after work today.

Reply:
Offset works great for people but for some reason when I add the offset it makes the hamsters to use the equipped-HELMET.png instead of the correct equipped-HELMET-hamster.png. I may resolve this by splitting the hat into a taller hat (the one that implements the offset) and a shorter hat that's wearable by hamsters (previous texture). Unless someone knows how to fix this?

Reply:
I implemented a hat with the offset (I used it to make the hat taller), this breaks the hamster texture so I left the old one without the offset so hamsters still have something to wear, they're now two complete hats. (I made the time requirement larger for the super tall hat as it's so tall as to be potentially disruptive so I wanted it to be rare.)